content/getting-started/multiDeviceMultiSig/_index.en.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Creating a self-managed, multi-device multisig wallet"
+date: 2022-02-22T16:03:11.470Z
+lastmod: 2022-02-22T16:03:11.470Z
+weight: 6
+draft: false
+# search related keywords
+keywords: ["mac, macos, windows, linux, ios, android"]
+---
+
+{{< notice note >}}
+This tutorial is for someone that would like their transactions to be signed from multiple devices. In this workflow, we will be using a desktop, a laptop, and a mobile phone to create a 2-of-3 multisig wallet. You can also use this workflow as a template to create more complex multisig wallets.
+{{< /notice >}}
+
+{{< notice info >}}
+You must be signed in to your account in order to access the messaging system. <br/>
+You will need to import or create a key for each of the devices you wish to use in the wallet. The keys cannot be the same. If you need help creating keys click <u>[here](/getting-started/createsoftwarekey/).</u>
+{{</ notice >}}
+
+{{< tabs >}}
+
+  {{< tab "Desktop" >}}
+  
+1. Open the chat feature located on the left side menu. <br/><br/>
+    
+<img src = desktopChatButton.png
+    alt = "Chat Button (Screenshot)"/>
+
+2. Click the "Start new message" button and type in your user email. You should see your name and email pop up below, click it!<br/><br/>
+
+<img src = desktopStartNewMessage.png
+    alt = "start new message button (Screenshot)"/>
+<img src = desktopNameSelected.png
+    alt = "Name selected (Screenshot)"/>
+
+3. Send a friendly message to yourself to initiate the chat room. After sending the message, you should see a new chat room with yourself!<br/><br/>
+
+<img src = desktopJoinedChatRoom.png
+    alt = "Wallet + button (Screenshot)"/>
+
+4. Click the "Create a collaborative wallet" button to initiate the multisig configuration setup.<br/>
+
+<img src = desktopCreateCollabWalletBttn.png
+    alt = "collaborative wallet question (Screenshot)"/>
+
+5. Name your wallet and click "Continue".<br/><br/>
+    
+<img src = desktopWalletName.png
+    alt = "Create collaborative wallet view (Screenshot)"/>
+
+6. Add the number of keys you will be using in your wallet. Then, add the required keys to sign a transaction.
```

Comment:
Specify the number of keys you will be using in your wallet. Then, specify the required number of keys needed to sign a transaction.